.github/pull_request_template.md

```diff
@@ -12,4 +12,12 @@ Resolves #
 
 ### Tests
```

Comment:
It might be helpful to note that it is only required to test one browser, since reviewers can help. Unless that rule has changed.